src/blr/cli/bamtofq.py

```diff
@@ -0,0 +1,96 @@
+"Writes fq files from bam files"
+
+import logging
+import pysam
+
+from tqdm import tqdm
+import dnaio
+
+from blr import utils
+
+logger = logging.getLogger(__name__)
+
+
+def main(args):
+    logger.info(f"Starting analysis")
+    query_cache = dict()
+    summary = Summary()
+    out_interleaved = not args.r2_out
```

Comment:
The interleaved options does not work as currently implemented. `r2_out`is a required positional argument. I suggest dropping interleaved FASTQ compatibility for now.

Reply:
Yeah I just forgot to remove this line.